src/styles/Detailspage.css

```diff
@@ -0,0 +1,110 @@
+.detailspage {
+  font-family: Arial, sans-serif;
+}
+
+.parent {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
+  margin-top: 30px;
```

Comment:
- [ ] Please remove the margin here and apply the layout colors as using your own color. You need only two colors and the color white as the font color.